packages/openapi-ts/src/plugins/@pinia-colada/sdk/config.ts

```diff
@@ -0,0 +1,21 @@
+import type { Plugin } from '../../types';
+import { handler } from './plugin';
+import type { Config } from './types';
+
+export const defaultConfig: Plugin.Config<Config> = {
+  _dependencies: ['@hey-api/typescript'],
+  _handler: handler,
+  exportFromIndex: true,
```

Comment:
Let's not export by default just yet since we can't guarantee the build won't crash due to naming conflicts. Let people select this manually if they're sure their build will pass